packer/resources/features/elk-stack/install.sh

```diff
@@ -18,7 +18,18 @@ sleep 1
 ## Update index and install packages
 apt-get update
 apt-get --yes --force-yes install ruby ruby-dev logstash elasticsearch=1.4.4 \
-    nodejs python-pip golang
+    nodejs python-pip golang libwww-perl libdatetime-perl nginx
+
+## Install Cloudwatch monitoring scripts
+wget "http://aws-cloudwatch.s3.amazonaws.com/downloads/CloudWatchMonitoringScripts-1.2.1.zip" -O /tmp/CloudWatchMonitoringScripts.zip
+unzip /tmp/CloudWatchMonitoringScripts.zip -d /tmp/
+rm /tmp/CloudWatchMonitoringScripts.zip
+mv /tmp/aws-scripts-mon /usr/local/
+
+# Add disk space cron job
+crontab -u root - << EOM
+*/5 * * * * /usr/local/aws-scripts-mon/mon-put-instance-data.pl --disk-space-util --mem-util --disk-path=/ --disk-path=/data --auto-scaling --from-cron
+EOM
```

Comment:
@theefer Can you link to where you install - should help andy see how feasible it is to extract the commonality. If everyone uses it in fundamentally different ways, then it might not be that helpful.